telco-core/configuration/reference-crs-kube-compare/metadata.yaml

```diff
@@ -188,6 +188,20 @@ parts:
         allOf:
           - path: optional/other/mount_namespace_config_master.yaml
           - path: optional/other/mount_namespace_config_worker.yaml
+  - name: optional-cert-manager
+    description: |-
+      Cert-manager operator for automated certificate management
+    components:
+      - name: cert-manager-operator
+        allOrNoneOf:
+          - path: optional/cert-manager/certManagerNS.yaml
+          - path: optional/cert-manager/certManagerOperatorgroup.yaml
+          - path: optional/cert-manager/certManagerSubscription.yaml
+          - path: optional/cert-manager/certManagerClusterIssuer.yaml
+          - path: optional/cert-manager/ingressCertificate.yaml
+          - path: optional/cert-manager/apiServerCertificate.yaml
+          - path: optional/cert-manager/apiServerConfig.yaml
+          - path: optional/cert-manager/ingressControllerConfig.yaml
```

Comment:
These should be a different category than allOrNoneOf. Perhaps separate the ingress pieces into a separate "allOrNoneOf" and api server into another one.